tests/testflows/rbac/tests/syntax/alter_row_policy.py

```diff
@@ -0,0 +1,244 @@
+from contextlib import contextmanager
+
+from testflows.core import *
+
+from rbac.requirements import *
+import rbac.tests.errors as errors
+
+@TestFeature
+@Name("alter row policy")
```

Comment:
I am not very good at python, could you help me understand what exactly the line like `@Name("alter row policy")` does?

Reply:
It is calling a decorator. A decorator is a function or an object that decorates some function. In this case these decorators decorate a function that implements the body of the test. So in this case we have

```python
@TestFeature
@Name("alter row policy")
```

where `@TestFeature` says that we are "defining" a `Feature` and the `@Name` will set the name of the `Feature` to `alter row policy`. Here are some references:

* https://testflows.com/handbook/#Decorated
* https://testflows.com/handbook/#Test-Definition-Classes
* https://www.programiz.com/python-programming/decorator

Reply:
Thanks for the explanation.